ts/src/abstractMachine/logger.ts

```diff
@@ -1,5 +1,8 @@
 import * as bunyan from "bunyan";
-export default bunyan.createLogger({
-    level: "fatal",
```

Comment:
Good thing your not on fatal status anymore!!!

Reply:
ikr!

Reply:
😌